pkg/ddevapp/config.go

```diff
@@ -518,10 +507,28 @@ func (app *DdevApp) docrootPrompt() error {
 	// Ensure the docroot exists. If it doesn't, prompt the user to verify they entered it correctly.
 	fullPath := filepath.Join(app.AppRoot, app.Docroot)
 	if _, err := os.Stat(fullPath); os.IsNotExist(err) {
-		output.UserOut.Errorf("No directory could be found at %s. Please enter a valid docroot\n", fullPath)
-		app.Docroot = ""
-		return app.docrootPrompt()
+		util.Warning("Warning: the provided docroot at %s does not currently exist.", fullPath)
+
+		// Ask the user for permission to create the docroot
+		for {
+			fmt.Printf("Create docroot at %s? [Y/n]: ", fullPath)
+			resp := util.GetInput("")
```

Comment:
util.Prompt() is just a wrapper on util.GetInput() that does the prompt also.

Reply:
Note that [Y/n] normally means that 'Y' is the default, but if you hit <enter> here, it just asks you again.

```
$ ddev config
You are reconfiguring the project at /tmp/junk.
The existing configuration will be updated and replaced.
Project name (junk):

The docroot is the directory from which your site is served. This is a relative path from your project root (/tmp/junk)
You may leave this value blank if your site files are in the project root
Docroot Location (docroot): junker99
Warning: the provided docroot at /tmp/junk/junker99 does not currently exist.
Create docroot at /tmp/junk/junker99? [Y/n]:
Create docroot at /tmp/junk/junker99? [Y/n]:
Create docroot at /tmp/junk/junker99? [Y/n]: y
Created docroot at /tmp/junk/junker99.
Found a php codebase at /tmp/junk/junker99.
Project Type [drupal6, drupal7, drupal8, wordpress, typo3, backdrop, php] (php):
Configuration complete. You may now run 'ddev start'.
```

Reply:
Good point - I agree that's how it should behave. I'll update this.